tools/docker/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Docker
+page: Installation
+section: tools
+---
+
+#**Getting started with Docker**
+
+###**Installing Docker on Fedora**
+
+####**Prerequisites**
+
+Docker is currently supported on the following versions of Fedora:
+
+* Fedora 20
+* Fedora 21
+* Fedora 22
+
```

Comment:
So why don't we just say that it's been supported since Fedora 20? That way, it would be unnecessary to update the page on every Fedora release.